pkg/cmd/roachtest/tests/versionupgrade.go

```diff
@@ -287,3 +290,69 @@ for i in 1 2 3 4; do
 done
 `)
 }
+
+func registerHTTPRestart(r registry.Registry) {
+	r.Add(registry.TestSpec{
+		Name:             "http-restart/mixed-version",
+		Owner:            registry.OwnerTestEng,
```

Comment:
nit: should test eng own this or obs? I'd argue that obs should own it since its testing that the http server is registering routes at the proper time.

Reply:
Above reasoning makes sense to me; +1 for obs.